org.kartkrew.RingRacers.metainfo.xml

```diff
@@ -0,0 +1,83 @@
+			are going go-karting! Use the untapped potential of rings to
+			super-charge your vehicle across more than 200 crazy courses.
+			With over 20 unique items and some of the hottest moves on this
+			side of the Floating Island, this is <b>Racing at the Next Level!</b>
+		</p>
+		<p>
+			Ring Racers is packed to the brim with features, building off the
+			success of its predecessor SRB2Kart, including:
+		</p>
+		<ul>
+			<li>A ludicrously long list of courses and playable characters</li>
+			<li>A complete Grand Prix mode with CPU opponents and multiple difficulty levels</li>
+			<li>New and revamped items</li>
+			<li>Unique game systems and mechanics</li>
+			<li>Local split-screen multiplayer up to 4 players, and online play up to 16</li>
+			<li>A comprehensive Time Attack mode with replay support</li>
+			<li>A wealth of secrets to find</li>
+		</ul>
+		<p>
+			In the Flatpak version of Ring Racers, your game data is located in
+			<code>~/.var/app/org.kartkrew.RingRacers/.ringracers</code>. In order
+			to load user modifications, please store them in this directory, or
+			override your Flatpak configuration.
+		</p>
+		<p>
+			Kart Krew Dev is not affiliated with SEGA Corporation. Dr. Robotnik's Ring
+			Racers is a work of fan art.
+		</p>
+	</description>
+	<launchable type="desktop-id">org.kartkrew.RingRacers.desktop</launchable>
+	<screenshots>
+		<screenshot type="default">
+			<image>https://raw.githubusercontent.com/flathub/org.kartkrew.RingRacers/master/images/screenshot_1.png</image>
+			<caption>Title screen</caption>
+		</screenshot>
+		<screenshot>
+			<image>https://raw.githubusercontent.com/flathub/org.kartkrew.RingRacers/master/images/screenshot_2.png</image>
+			<caption>Dr. Robotnik drifting in Robotnik Coaster Zone</caption>
+		</screenshot>
+		<screenshot>
+			<image>https://raw.githubusercontent.com/flathub/org.kartkrew.RingRacers/master/images/screenshot_3.png</image>
+			<caption>Dr. Robotnik aiming at UFOs in CD Special Stage 1</caption>
+		</screenshot>
+		<screenshot>
+			<image>https://raw.githubusercontent.com/flathub/org.kartkrew.RingRacers/master/images/screenshot_4.png</image>
+			<caption>Sonic and Tails neck-and-neck in Sonic Speedway Zone</caption>
+		</screenshot>
+		<screenshot>
+			<image>https://raw.githubusercontent.com/flathub/org.kartkrew.RingRacers/master/images/screenshot_5.png</image>
+			<caption>Dr. Robotnik hits an unsuspecting Tails with an item in Sonic Speedway Zone</caption>
+		</screenshot>
+	</screenshots>
+	<url type="homepage">https://www.kartkrew.org</url>
+	<url type="bugtracker">https://git.do.srb2.org/KartKrew/RingRacers/-/issues</url>
+	<developer_name>Kart Krew Dev</developer_name>
+	<provides>
+		<binary>ringracers</binary>
+	</provides>
+	<releases>
+		<release version="2.0" date="2024-04-24">
+			<description>
+				<p>Initial release.</p>
+				<p>See the release article: https://www.kartkrew.org/news/2024-04-24_dr-robotniks-ring-racers-version-20.html</p>
```

Comment:
bare URL are not allowed either. There is an `<url> ctag that can be used for that.

Reply:
I'm looking at the docs for appstream and I see the whitelisted tags, but not the ` thing. What is that?

Reply:
`<url>`